client/git_token.go

```diff
@@ -3,7 +3,7 @@ package client
 type GitToken struct {
 	Id             string `json:"id"`
 	Name           string `json:"name"`
-	Value          string `json:"value"`
+	Value          string `json:"value" tfschema:"-"`
```

Comment:
can you send a doc / explanation as to why setting a field's schema to `-` works? 

tried to find an explanation [here](https://github.com/minamijoyo/tfschema) but couldn't really find any

Reply:
tfschema is something internal we wrote.
See https://github.com/env0/terraform-provider-env0/blob/main/DEVELOPMENT.md

`Pay attention to the following caveats:...`

Reply:
> tfschema is something internal we wrote.

Oh i thought this refers to this https://github.com/minamijoyo/tfschema